namecheap/resource_namecheap_record.go

```diff
@@ -22,6 +25,13 @@ func resourceNameCheapRecord() *schema.Resource {
 		Read:   resourceNameCheapRecordRead,
 		Delete: resourceNameCheapRecordDelete,
 
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(ncDefaultTimeout * time.Second),
+			Update: schema.DefaultTimeout(ncDefaultTimeout * time.Second),
+			Read:   schema.DefaultTimeout(ncDefaultTimeout * time.Second),
+			Delete: schema.DefaultTimeout(ncDefaultTimeout * time.Second),
+		},
+
 		Schema: map[string]*schema.Schema{
 			"domain": &schema.Schema{
```

Comment:
BTW My IDE says these are redundant, maybe we can remove these to top level nodes?

Reply:
Maybe? I probably copied the code structure from previous Terraform providers. Feel free to condense the code if you want.